vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -230,6 +233,16 @@ def parse_container_port_config(port_config, pod_id, container_name)
       }
     end
 
+    def parse_service_port_config(port_config, service_id)
+      {
+        :ems_ref     => "#{service_id}_#{port_config["port"]}_#{port_config["targetPort"]}_#{port_config["protocol"]}",
```

Comment:
> Maybe in this case ems_ref can be omitted altogether:
> 
>     service ports are not an entity on their own (they are just a one-to-many items of services)
>     using all the attributes as ems_ref basically means that there's no field that can be updated without generating a new entity
> 
> If save_inventory_multi supports it I'd suggest to remove the ems_ref in this specific case. (/cc @Fryguy )

this is similar to what was done in containers/containers def/container port configs - they all don't have a "real" ems_ref. iirc, it was advised by @Fryguy to have those ems_ref fields generated (@Fryguy pls correct me if I'm wrong).
I'm OK to drop them if they are not needed. And if we're gonna drop them -imo  it has to be done for all the "nested" entities.